src/routes/[type=mediaType]/[id]/[season]/+page.svelte

```diff
@@ -1,11 +1,81 @@
 <script lang="ts">
 	import type { PageData } from './$types';
 	import Header from '$lib/components/header.svelte';
-	import { Star } from 'lucide-svelte';
+	import { Star, CirclePower, RotateCcw } from 'lucide-svelte';
 	import { formatDate } from '$lib/helpers';
 	import { statesName } from '$lib/constants';
+	import { ItemsService } from '$lib/client';
+	import { toast } from 'svelte-sonner';
+	import { invalidateAll } from '$app/navigation';
 
 	export let data: PageData;
+
+	let selectedEpisodeNumber: number | null = null;
+	let isResetting = false;
+	let isRetrying = false;
+
+	async function handleMediaAction(
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Please define the types

Reply:
Problem is that I'm not sure what it return. I would have to do union types here. But I can do that.